tests/test_part_image.py

```diff
@@ -337,21 +344,21 @@ def test_equal_checksum_part_image_and_artifact(self, latest_part_image, latest_
                     break
                 hash.update(buf)
             artifact_hash = hash.hexdigest()
-            artifact_ls = subprocess.check_output(["ls", "-l", tmp_artifact.name])
+            artifact_ls = subprocess.check_output(["ls", "-l", tmp_artifact.name]).encode()
```

Comment:
I don't understand this encoding here. `check_output` is already returning bytes, right?

Reply:
I guess you meant to have `.decode()` here to be able to format the string below

Reply:
Yes, I also wondered about this. I think this applies to the next `encode` below as well.

Reply:
Well spotted. Of course it should be decoded, not encoded. Fixed now!